form-view-model/src/main/kotlin/com/ritense/formviewmodel/SpringContextHelper.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.formviewmodel
+
+import mu.KotlinLogging
+import org.springframework.context.ApplicationContext
+import org.springframework.context.ApplicationContextAware
+
+class SpringContextHelper : ApplicationContextAware {
```

Comment:
This class seems to be unused. Why is it there?

Reply:
It is used when I look at it in intelliJ 

```
fun <T> dispatchCommand(command: Command<T>) =
    SpringContextHelper.getBean(CommandDispatcher::class.java).dispatch(command)
```

Reply:
That method is pointing towards the `com.ritense.commandhandling.SpringContextHelper`. Not this one.

Reply:
Will remove